queries/php/textobjects.scm

```diff
@@ -25,3 +25,6 @@
 ;; parameters
 (formal_parameters
   (simple_parameter) @parameter.inner)
+
+(arguments
+  (argument) @parameter.outer)
```

Comment:
I think we are marking these as `@parameter.inner`, outer would be including `,` separators

Reply:
I'm not quite following you, can you explain a bit more.

Reply:
You need to change this to `@parameter.innner`, outer is used when the capture includes the argument separator/whitespaces.

Reply:
Thanks for being more clear :+1: